pkg/parsers/bitbucket/jobs.go

```diff
@@ -143,6 +141,17 @@ func parseExecutionUnitToStep(executionUnitRef *bitbucketModels.ExecutionUnitRef
 	return &step
 }
 
+func setStepType(step *models.Step) models.StepType {
```

Comment:
`setStepType` should set the step type, this function should be called `calculateStepType` or `getStepType`